src/components/Concentrator/ConcentratorRewards.tsx

```diff
@@ -63,7 +72,10 @@ export const ConcentratorRewards: FC<ConcentratorRewardsProps> = ({
         <dl className="grid grid-cols-[1fr,auto] gap-y-2">
           <dt className="text-xs font-medium text-white/80">AUM</dt>
           <dd className="text-right text-xs font-medium text-white/80">
-            $3,067,000
+            <ConcentratorRewardsAum
```

Comment:
I'm not sure about the AUM here, what's the difference with the concentrator TVL - need more clarification here.

Reply:
I believe it's the same thing